assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -253,288 +253,277 @@ db.schema.flyway.scripts.prefix=
 # Suffix of migration scripts.
 db.schema.flyway.scripts.suffix=.sql
 
-# Separator of version from the other part of script name.
+# Separator of `version` from the other parts of script name.
 db.schema.flyway.scripts.version_separator=__
 
-# Locations where to search migration scripts.
+# Locations where to search for migration scripts.
 db.schema.flyway.scripts.locations=classpath:che-schema
 
-### Kubernetes Infra parameters
+### Kubernetes infrastructure parameters
 
-# Configuration of Kubernetes client master URL that Infra will use.
+# Configuration of Kubernetes Client master URL that the infrastructure uses.
 che.infra.kubernetes.master_url=
 
-# Boolean to configure Kubernetes client to use trusted certificates.
+# Boolean to configure Kubernetes Client to use trusted certificates.
 che.infra.kubernetes.trust_certs=false
 
-# Kubernetes cluster domain. If not set, svc names will not contain information about the cluster domain.
+# Kubernetes cluster domain. If not set, svc names doesn't contain information about the cluster domain.
 che.infra.kubernetes.cluster_domain=NULL
 
-# Defines the way how servers are exposed to the world in Kubernetes infra.
-# List of strategies implemented in {prod-short}: `default-host`, `multi-host`, `single-host`.
+# Defines how servers are exposed to the world in Kubernetes infrastructure.
+# List of strategies implemented in {prod-short}: `default-host`, `multi-host`, and `single-host`.
```

Comment:
Use `Possible values are:` instead of `List of ...` to be consistent with other lists of options.

Reply:
Also this property might be obsolete -- @tolusha would know if we still support all these options.

Reply:
Obsolete and can be removed